packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -286,15 +311,32 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 				};
 			},
 			configurePreviewServer(vitePreviewServer) {
+				const workerConfigs = getWorkerConfigs(vitePreviewServer.config.root);
+
 				const miniflare = new Miniflare(
 					getPreviewMiniflareOptions(
 						vitePreviewServer,
-						pluginConfig.persistState ?? true
+						workerConfigs,
+						pluginConfig.persistState ?? true,
+						pluginConfig.inspectorPort
 					)
 				);
 
+				if (workerConfigs.length >= 1) {
+					addDebugToVitePrintUrls(vitePreviewServer);
+				}
+
+				const workerNames = workerConfigs.map((worker) => worker.name ?? "");
```

Comment:
What does it mean for a worker to have no name here?

Reply:
mh.... I need to double check, in miniflare you can provide workers without names, but I actually think that the name is mandated in `readConfig`? so it might be that the name is always defined? :thinking: